packages/next-codemod/bin/__testfixtures__/next-15-canary-44-installed/README.md

```diff
@@ -0,0 +1 @@
+does not suggest `next-dynamic-access-named-export` transform
```

Comment:
Suggested update before. fixtures have a separate commit so previous behavior can be checked.